lib/backend/syncersv1/updateprocessors/rules.go

```diff
@@ -153,6 +154,14 @@ func parseNamespaceSelector(s string) string {
 	return updated
 }
 
+// normalizeProto converts a v1 protocol string to a v3 protocol string
```

Comment:
Doesn't this convert a camelcase v3 protocol to a v1 lowercase protocol?

Maybe normalize is not the correct word - after all, what is normal in this case?

Reply:
A profound question, worthy of a lengthy dissertation. 
Renamed to `convertV3ProtocolToV1`